tensorflow_datasets/core/features/bounding_boxes.py

```diff
@@ -79,6 +82,38 @@ def encode_example(self, bbox):
         [bbox.ymin, bbox.xmin, bbox.ymax, bbox.xmax]
     )
 
+  def _build_thumbnail_with_bbox(self, ex: np.ndarray):
```

Comment:
As `self` isn't used anywhere, I would make this a simple function rather than method